src/v/io/clang-tidy-helper.cc

```diff
@@ -0,0 +1,5 @@
+#include "io/cache.h"
+#include "io/interval_map.h"
+#include "io/page.h"
+#include "io/page_set.h"
+#include "io/persistence.h"
```

Comment:
What does this help with? Maybe add a comment or a mention in the commit message? Also does it need a license header?

Reply:
thanks for flagging. that was part of a follow-up PR (which explains the lint-helper) that snuck in via a sloppy rebase-fixup.